docs/src/main/paradox/scala/http/sse-support.md

```diff
@@ -0,0 +1,57 @@
+<a id="sse-support-scala"></a>
```

Comment:
I don't think id's need to be unique across java and scala versions anymore - actually might be better to have them the same because we want to eventually merge them.

We can avoid some duplication by making the markdown in the 'scala' folder support both java and scala (using paradox groups, http://developer.lightbend.com/docs/paradox/latest/features/groups.html), then the file in the 'java' folder can simply be a symlink. Not necessary for this PR though IMO, looks good!